CMake/global_config.cmake

```diff
@@ -23,6 +23,12 @@ macro(global_set_flags)
 
     add_definitions(-DELPP_THREAD_SAFE)
     add_definitions(-DELPP_NO_DEFAULT_LOG_FILE)
+    
+    #Performance improvement with Ubuntu 18/20
+    if(UNIX AND (NOT ANDROID_NDK_TOOLCHAIN_INCLUDED))
+       message(INFO "Asynchronous ELPP invoked")
+       add_definitions(-DELPP_EXPERIMENTAL_ASYNC)
```

Comment:
I don't like "EXPERIMENTAL" in there... it'll eventually be non-experimental.

Reply:
It also doesn't fit with `ELPP_ASYNC_LOGGING` below...

Reply:
The flag ELPP_EXPERIMENTAL_ASYNC is from the easylogging original repo.
In my opinion, it seems like a good idea to keep the same flags so that we could see the differences with it when we want to.

Reply:
Si what is `ELPP_ASYNC_LOGGING`?

Reply:
This is an inner flag in the code.
Part of their code:
#if defined(ELPP_EXPERIMENTAL_ASYNC)
#  define ELPP_ASYNC_LOGGING 1
#else
#  define ELPP_ASYNC_LOGGING 0
As you see, the flags are the same. I can delete the experimental one if you want.

Reply:
flag added